graphqlapi/src/main/java/io/stargate/graphql/web/StargateGraphqlContext.java

```diff
@@ -53,6 +53,8 @@
   // For more information.
   private final BatchContext batchContext = new BatchContext();
 
+  private volatile boolean overloaded;
```

Comment:
Maybe add brief Javadoc here (or on setter) mentioning reason; flag to indicate that overload fail (429) was received during fetching.

Reply:
I can not find this new `volatile` anywhere, is this removed or what s going on?

Reply:
Volatile is enough, we just need to guarantee that if the value was changed, the change will be visible.

We don't need synchronization because:
* the only thing writers can do is set the value to `true`. So even if this happens multiple times, they don't race with each other. (PS: it can happen multiple times, if a GraphQL payload contains multiple operations and more than one `OverloadedException` occurs)
* we read back the value after the execution of the payload (`GraphqlResourceBase`). All exception handlers are guaranteed to have run at that point, since their output is part of the response.